broker/test/test_broker/eventmesh.LockManager.spec.js

```diff
@@ -328,16 +328,17 @@ describe('eventmesh', () => {
       it('should fail to unlock resource after multiple retries', () => {
         const payload1 = {
           metadata: {
-            resourceVersion: samplelock6.metadata.resourceVersion
+            resourceVersion: samplelock4.metadata.resourceVersion
           }
```

Comment:
As `samplelock6.metadata.resourceVersion` is undefined, it was creating an issue when we added the code to strictly match the payload while updating the nock version. Due to this reason, the mock for the `PATCH` request that we have created does not match with the actual request. Hence, I have changed the `samplelock` in which the `resourceVersion` was well defined.